### DISCUSSION TITLE
peak2gene

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#690] [Questions / Documentation]
Hello again,
I am interested in using peak2gene to find potential enhancers. I wanted to get your expert opinion on how far the peak could be. For example, I have a peak that is 162449bp away from the gene that it's correlated to (and it has a good correlation of 0.7). In your opinion, should I ignore it since it's too far (given that some enhancers could be really far)
Thank you!

**Comment by rcorces** [comment]
I would not exclude this. 162 kb is long but still very much in the realm of possibility. Without some orthogonal information, its impossible to know if it is real or not, but I wouldnt exclude it based on distance.
